Schema/Blueprint.php

```diff
@@ -203,7 +203,7 @@ public function dropIfExists()
     /**
      * Indicate that the given columns should be dropped.
      *
-     * @param  string|array  $columns
+     * @param  array|mixed  $columns
```

Comment:
```mixed``` technically means "everything" already - what is the point of ```array``` here?! You should remove one of them, depending on the workflow of these methods.